client/webserver/locales/en-us.go

```diff
@@ -95,9 +95,10 @@ var EnUS = map[string]string{
 	"Settled":                        "Settled",
 	"Status":                         "Status",
 	"view order history":             "view order history",
-	"cancel order":                   "cancel order",
+	"cancel_order":                   "cancel order",
 	"order details":                  "order details",
 	"verify_order":                   `Verify <span id="vSideHeader"></span>  Order`,
+	"prevent_temporary_overlocking":  "Prevent temporary over locking of funds",
```

Comment:
I realize we're sorta making a new word with "overlocking", but I would prefer that or "over-locking" to two words.